### PR TITLE
Update create-a-phone-system-auto-attendant.md

### PR DESCRIPTION
**Pull request by CLYVR**
Updating information on the AA to Voicemail options to identify existing transcription option and new suppress greeting option.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-auto-attendant.md [hunk 1]
```diff
@@ -42,6 +42,11 @@ Auto attendants can direct calls, based on callers' input, to one of the followi
 
 You'll be prompted to choose one of these options at various stages as you set up an auto attendant.
 
+> [!NOTE]
+> When choosing Voicemail as a destination, two additional options are available:
+> - **Transcription** - (Default: Off) When enabled, the voicemail message will be transcribed and included as part of the email
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:47]:
```suggestion
> - **Transcription** (Default: Off): When enabled, the voicemail message will be transcribed and included as part of the email.
```

File: Teams/create-a-phone-system-auto-attendant.md [hunk 2]
```diff
@@ -42,6 +42,11 @@ Auto attendants can direct calls, based on callers' input, to one of the followi
...
+> - **Suppress Greeting** - (Default: Off) When enabled, the standard system message "Please leave a message after the tone. When yo have finished please hang up or press the hash key for more options." will be suppressed.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:48]:
```suggestion
> - **Suppress Greeting** (Default: Off) - when enabled, the standard system message "Please leave a message after the tone. When you have finished please hang up or press the hash key for more options." will be suppressed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**